docs/releases/ssdlc-compliance.template.md

```diff
@@ -0,0 +1,30 @@
+SSDLC Compliance Report: Atlas CLI ${VERSION}
+=================================================================
+
+- Release Creator: ${AUTHOR}
+- Created On:       ${DATE}
+
+Overview:
+
+- **Product and Release Name**
+    - Atlas CLI ${VERSION}, ${DATE}.
+
+- **Process Document**
+  - http://go/how-we-develop-software-doc
+
+- **Tool used to track third party vulnerabilities**
+  - [Kondukto](https://arcticglow.kondukto.io/)
+
+- **Dependency Information**
+  - See SBOM Lite manifests (CycloneDX in JSON format):
+    - https://github.com/mongodb/mongodb-atlas-cli/releases/download/atlascli%2F${VERSION}/sbom.json
```

Comment:
different from K8s, who publishes it in the code, we'll link to the release artifact